### PR TITLE
Add and document and explicit `toARGB32`.

### PR DESCRIPTION
**Pull request by matanlurey**
Closes https://github.com/flutter/flutter/issues/157128.

### CODE REVIEW

File: testing/dart/color_test.dart [hunk 1]
```diff
@@ -339,6 +339,11 @@ void main() {
     // Call base class member, make sure it uses overridden value.
     expect(color.red, 0xE0);
   });
+
+  test('value and toARGB32 are equivalent methods', () {
+    const Color halfRed = Color.from(alpha: 1.0, red: 0.5, green: 0, blue: 0);
+    expect(halfRed.value, halfRed.toARGB32());
```
Comment (gaaclarke) [inline testing/dart/color_test.dart:345]: It's probably better not to introduce a new test that uses a deprecated property.  I'd just assert it is a specific integer.

**Review by gaaclarke — APPROVED** [review]
lgtm, nit about adding test that uses deprecated member.

File: testing/dart/color_test.dart (hunk 1 above)
Comment (matanlurey) [reply testing/dart/color_test.dart]: Sounds good!

**Comment by matanlurey** [comment]
Got my local test runner working again, and I'll have this fixed and merged shortly.

**Pull request merged**